packages/iql/package.json

```diff
@@ -21,7 +21,7 @@
         "build:node": "npm run build",
         "build:parser": "peggy -o ./src/iql.gen.js ./src/grammar.peggy",
         "build:watch:parser": "npx nodemon -w src/grammar.peggy --exec 'npm run build:parser'",
-        "demo": "npm run build && light-server -s ./demo -w 'src/grammar.peggy, demo/app.js, demo/attributes.js, demo/index.html # cd demo && npm run build && webpack --mode production'",
+        "demo": "npm run build && npx webpack --mode production --config ./demo/webpack.config.js && light-server -s ./demo -w 'src/grammar.peggy, demo/app.js, demo/attributes.js, demo/index.html # npx webpack --mode production --config ./demo/webpack.config.js'",
```

Comment:
why is there a comment in this script?

Reply:
Nah, that's the command `light-server` will run when any of the watched files change. That command is separated by a hash inside the command string.

Reply:
nvm, it's the execution script for the light-server watch, right?